src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -772,6 +772,151 @@ def hcells_list(self, value: List[str]) -> None:
     ### END Generated interface HammerLVSTool ###
 
 
+# TODO(daniel)
+class HammerSimTool(HammerTool):
+  
+    #@abstractmethod
+    #def fill_outputs(self) -> bool:
+    #  pass
+
+    # export_config_outputs not necessary because not passing anything downstream?
+
+    @property
+    def input_files(self) -> List[str]:
```

Comment:
Please use `generate_properties.py` for this rather than manually writing these getters/setters